### ISSUE TITLE
Fehlerhafte Darstellung der Icons bei Rebsorten

### ISSUE THREAD

**Issue by vvvika** [zentrumnawi/solid-frontend#2010]
Hallo an das S.O.L.I.D-Team,
ich weiß nicht so ganz, ob es sich um ein Frontend- oder Backend-Fehler handelt, platziere es aber vorerst in diesem Bereich hier:

Bei den Rebsorten hat sich die Reihenfolge der Bilder wohl insofern geändert, als dass die Audiofiles nun als erste einsortiert werden. Das hat zur Folge, dass auch die Icons neben dem Namen im der Baumübersicht alle das Icon des Kopfhörers haben. Das sollte aber so nicht sein, oder?

![grafik](https://github.com/zentrumnawi/solid-frontend/assets/95678980/959ea481-f533-47f4-af89-e36c1c2bb561)

🍃

Gruß,
Viktoria

**Comment by sheepyhollow** [comment]
Ich habe das Problem identifiziert - durch den letzten Backend-Release hat sich die Reihenfolge der Medienobjekte in der Schnittstellen-Ausgabe verändert und der Thumbnail ist immer das erste Element der Medienobjekt-Liste. Bei vielen Reben ist das erste Element jetzt ein Audiofile weshalb das Audio-Icon als Thumbnail genommen wird.

Tatsächlich wird bei der Eingabe der Medien-Objekte auch eine Reihenfolge angegeben (Profile Position), die aber offenbar nicht richtig verarbeitet wird.

Es ist nicht schwer zu reparieren, ABER ich müsste wissen, wie Ihr genau festlegen wollt, welches Bild im Thumbnail stehen soll.
Das einfachste wäre, dass immer das Media-Objekt mit der niedrigsten `Profile Position` als Thumbnail gewählt wird.

Passt das so @vvvika ?

**Comment by vvvika** [comment]
Ja, ich denke, das ist korrekt so: Also dass ein  Media/Image?-Objekt mit der niedrigstes (also unter "1" eingetragenen) Profile Position als Thumbnail genommen werden sollte. 
![grafik](https://github.com/zentrumnawi/solid-frontend/assets/95678980/5c5496a0-f016-4e6e-8336-7b4949364269)

**Comment by sheepyhollow** [comment]
Genau das. Profile Position = 1 ist der thumbnail.

Patch ist unterwegs!

**Comment by sheepyhollow** [comment]
Fix is basically in [29d4770](https://github.com/zentrumnawi/solid-frontend/pull/1996/commits/29d47708e5d4556f5e5a18f55f2472f86c6bba36) and needs to be added after the framework upgrade.
